### ISSUE TITLE
Problem about relation in notion

### ISSUE THREAD

**Issue by jianyubej** [dvanoni/notero#44]
Hello there! I would like to thank you for the great work on notero. It is very helpful！ However, I met a problem about relation. If I relate the database of "notero" to another, for example, "Task", I can't find out any existing page of  "notero" in the property named like "relation to notero" in “Task”. And like the property named "Related References" in " notero", I can't find out other existing pages. So I can't relate a page to another page in "notero" or to another page in another database. I was wondering if there is a bug about creating a new page from zotero to notion and it lets the page be "unsearchable" or if there is sth wrong with the database template you were given. I would appreciate it if you can solve the doubts :)
Thank you!

<img width="1101" alt="iShot2022-02-08 23 50 31" src="https://user-images.githubusercontent.com/95463401/153024140-eaa1772b-8922-4849-82e8-8f0ea8432dea.png">

<img width="1042" alt="iShot2022-02-08 23 50 46" src="https://user-images.githubusercontent.com/95463401/153024174-91be3e20-6bf2-4065-995f-5fb1628c1ad8.png">

**Comment by dvanoni** [comment]
@jianyubej, thanks for using Notero!

I believe I've run into this same thing that you're seeing, and I think it's just how the relation properties work in Notion. As far as I know, they won't show any results for pages to link to unless you start typing a search query.

If you start typing the name of one of the pages that was synced by Notero, does it show up as a result?

**Comment by dvanoni** [comment]
Actually, I just played around in Notion a little more, and I think you can get results to show up by clicking in the relation field again after you open it up. You might not have to start a search after all.

https://user-images.githubusercontent.com/299357/153139560-3d65d387-3851-43a1-bf8e-404146a358e5.mp4

**Comment by jianyubej** [comment]
Yes, it works! I can get the right relation! Thank you for the help. And thanks again for the fantabulous work on notero. Sincere wishes to you in the distance. May everything go smoothly and happily with you :)
